src/components/AdobeTab.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
```

Comment:
Small comment: this is named `AdobeTab` and makes sense but the charts or list components are not named with `Tab`.